Jenkinsfile

```diff
@@ -7,22 +7,11 @@ pipeline {
         UACLIENT_BEHAVE_JENKINS_CHANGE_ID = "${CHANGE_ID}"
         UACLIENT_BEHAVE_BUILD_PR=1
         UACLIENT_BEHAVE_CONTRACT_TOKEN = credentials('ua-contract-token')
-        UACLIENT_BEHAVE_AWS_ACCESS_KEY_ID = credentials('ua-aws-access-key-id')
-        UACLIENT_BEHAVE_AWS_SECRET_ACCESS_KEY = credentials(
-            'ua-aws-secret-access-key'
-        )
-        UACLIENT_BEHAVE_AZ_CLIENT_ID = credentials('ua-azure-client-id')
-        UACLIENT_BEHAVE_AZ_CLIENT_SECRET = credentials(
-            'ua-azure-client-secret'
-        )
-        UACLIENT_BEHAVE_AZ_TENANT_ID = credentials('ua-azure-tenant')
-        UACLIENT_BEHAVE_AZ_SUBSCRIPTION_ID = credentials(
-            'ua-azure-subscription-id'
-        )
+        UACLIENT_BEHAVE_CLOUD_CREDENTIALS_PATH = credentials('pycloudlib-config')
         UACLIENT_BEHAVE_CONTRACT_TOKEN_STAGING = credentials(
             'ua-contract-token-staging'
         )
-        UACLIENT_BEHAVE_CONTRACT_TOKEN_STAGING_EXPIRED = credentials(
+        PYCLOUDLIB_CONFIG = credentials(
             'ua-contract-token-staging-expired'
```

Comment:
Wrong credential here?

Reply:
@lucasmoura ^^ that should be `pycloudlib-config` I believe, or maybe we can just drop `PYCLOUDLIB_CONFIG` and go with `UACLIENT_BEHAVE_CLOUD_CREDENTIALS_PATH`. If we're keeping `PYCLOUDLIB_CONFIG` then it should be added to `passenv` in `tox.ini` otherwise the test environment will never see the env var.